test/plots/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <head>
-  <meta charset="utf-8">
```

Comment:
Odd, not sure why this change was necessary. Will take a look tomorrow

Reply:
I've had the same experience on a different branch. Could it be that the github action has changed?

Reply:
put this tiny change in a separate PR #1059

Reply:
thank you! I'll merge main in once that's in